app/services/invite_schools.rb

```diff
@@ -221,6 +221,15 @@ def send_induction_coordinator_add_participants_email
     end
   end
 
+  def send_year2020_invite_email
+    return unless FeatureFlag.active?(:year_2020_data_entry)
```

Comment:
Are they planning a phased rollout of this? I really hope so....

Reply:
Maybe? I imagine it will end up being phased, and this is just a first iteration that we can build on when we know more requirements.

Reply:
The exact comms plan and email content will be done next sprint, apparently. So this will be improved upon - for now it's more of a foundation to make the future work easier.